apps/pie-docs/src/accessibility/accessibility-overview.md

```diff
@@ -0,0 +1,82 @@
+---
+eleventyNavigation:
+    key: Accessibility overview
+    parent: Accessibility
+    order: 1
+title: Accessibility overview
+description: In this section you’ll find useful information about accessibility and why it is important at JustEatTakeaway.com
+---
+
+## Introduction
+
+Welcome to the **Accessibility** section of PIE design system’s documentation site. Our goal is to make our platforms better for everyone by creating inclusive and user-friendly products. We are committed to accessibility based on the principles of universal design and inclusivity.
+
+{% contentPageImage {
+    src:"../../../assets/img/accessibility/accessibility-overview/accessibility-introduction.svg",
+    mobileSrc:"../../../assets/img/accessibility/accessibility-overview/accessibility-introduction_narrow.svg",
+    caption: "Inclusive design is better design for everyone, no user should be excluded because of disability."
+} %}
+
+---
+
+## Why is accessibility important?
+
+Just as a good recipe caters to the diverse tastes and dietary needs of all diners, a design system that incorporates accessibility ensures that all users, **regardless of their abilities or disabilities**, can access and use digital products and services.
+
+Incorporating accessibility into a design system can lead to a **better user experience, increased engagement**, and **increased customer loyalty**.
```

Comment:
I might be wrong here but I think when using bold on words you're meant to apply it to **each**, **individual**, **word** and **not** the comma 😄

Reply:
Agreed, thanks for noticing this!

Reply:
👍